R/map.R

```diff
@@ -23,6 +23,7 @@
 #' @inheritParams as_mapper
 #' @param .x A list or atomic vector.
 #' @param ... Additional arguments passed on to the mapped function.
+#' @param .progress Whether to show a progress bar.
```

Comment:
If `TRUE`, will only show a progress bar if deemed necessary, using some heuristics to ensure that very short tasks don't generate a progress bar.

Reply:
I added a bullet for better docs.